src/base-styles.css

```diff
@@ -54,3 +54,9 @@ body {
   user-select: none;
   -webkit-user-select: none;
 }
+
+input[type="number"]::-webkit-inner-spin-button,
+input[type="number"]::-webkit-outer-spin-button {
+  -webkit-appearance: none;
```

Comment:
What is the `-webkit-appearance: none` good for?

Reply:
Apparently it can be used to hide the buttons. I used it because it is mentioned in the examples on how to hide them and is supported on Chrome and Safari.